internal/nfs/nodeserver/nodeserver.go

```diff
@@ -200,7 +200,7 @@ func (ns *NodeServer) mountNFS(
 		err    error
 	)
 
-	notMnt, err := ns.Mounter.IsLikelyNotMountPoint(mountPoint)
+	notMnt, err := ns.Mounter.IsMountPoint(mountPoint)
```

Comment:
we should change `notMnt` to `Ismnt` or something like that and we should reverse the check for it because `IsLikelyNotMountPoint` was returning its a mount point or not as `!` operation of IsMountPoint

Reply:
Ah, yes :facepalm:

Reply:
This should have been addressed now. Thanks for catching!